### ISSUE TITLE
Means of input and output REQ 4 [NAUR]

### ISSUE THREAD

**Issue by JWJPaton** [w3c/apa#151]
The base requirement is that the device/service is accessible so we should be clear that a non-natural language alternative that fulfils an accessibility need is acceptable. 

I suggest changing
"REQ 4:Support multiple output devices and methods."
to 
"REQ 4: Support multiple output devices and methods either within the natural language interface/s or via alternatives."

**Comment by RealJoshue108** [comment]
@JWJPaton I've added these suggested edits to this branch

**Comment by RealJoshue108** [comment]
merged into main
